### DISCUSSION TITLE
Should we release 0.71.3?

### DISCUSSION THREAD

**Discussion by kelset** [reactwg/react-native-releases#57] [Patches]
### Released, please refer to https://github.com/reactwg/react-native-releases/discussions/58

---

Current Release: [0.71.2](https://github.com/facebook/react-native/releases/tag/v0.71.2)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1. ~(27/01/2023 13:19) https://github.com/facebook/react-native/commit/a00cea46bf0bde788e6f92ae8f750bd9e32a2739 (will likely require more commits cc @cipolleschi)~
2. ~(07/02/2023 15:29) https://github.com/facebook/react-native/commit/bb02ccf13f76f46b8572e2a85d578fd8d4fd9467~
3. ~(07/02/2023 17:33) https://github.com/facebook/react-native/commit/a232decbb1252ade0247a352f887ca4d97ee273c~

#### Local commits to backport to main

1.

**Comment by kelset** [comment]
once this is addressed, we'll have to cherry-pick the bump asap: https://github.com/react-native-community/jsc-android-buildscripts/issues/178

**Reply by kelset** [reply]
PR: https://github.com/facebook/react-native/pull/36062

**Reply by kelset** [reply]
commit: https://github.com/facebook/react-native/commit/a232decbb1252ade0247a352f887ca4d97ee273c

**Comment by kelset** [comment]
https://github.com/facebook/react-native/commit/bb02ccf13f76f46b8572e2a85d578fd8d4fd9467

**Reply by cortinico** [reply]
This also needs a bump of RNGP

**Comment by birkir** [comment]
https://github.com/facebook/react-native/commit/0a4dcb0309fdc8f4529ed7599c4170341b42c9b1

**Reply by kelset** [reply]
we won't cherry-pick this as it's not a fix but an addition; moreover, [see comments](https://github.com/facebook/react-native/pull/35989#discussion_r1101016329)

**Comment by kelset** [comment]
https://github.com/facebook/react-native/pull/36096

**Reply by tido64** [reply]
Merged directly to `0.71-stable`, so no cherry-pick is necessary.

**Comment by kelset** [comment]
https://github.com/facebook/react-native/commit/0487108461010154cf959bb0bf6ba9e82fdcc4d1

**Reply by cipolleschi** [reply]
This fix brings with him the bump to AGP 7.4.1 (https://github.com/facebook/react-native/commit/5647d79dc97ab2787a9575cb1621725d865b9814) so we will leave this out.

**Comment by gabrieldonadel** [comment]
Should we cherry-pick https://github.com/facebook/react-native/commit/8b1f6e09c1b11e07096156d8d01aaff649745fbc?

Seems that some users are facing `Excessive number of pending callbacks:` pretty often

**Reply by kelset** [reply]
sorry Gabriel, by the time you posted this comment we already did the cherry-picking and testing for this patch (we're releasing it now); we'll add this commit to the next one

**Reply by gabrieldonadel** [reply]
No worries, thanks @kelset
